src/containers/MyNdla/ArenaCards/ArenaCard.tsx

```diff
@@ -0,0 +1,218 @@
+import SafeLink from '@ndla/safelink';
+import {
+  colors,
+  spacing,
+  breakpoints,
+  mq,
+  fonts,
+  misc,
+  spacingUnit,
+} from '@ndla/core';
+import Icon from '@ndla/icons';
+import { Forum, ForumOutlined, Locked } from '@ndla/icons/common';
+import { formatDateTime } from '../../../util/formatDate';
+
+interface Props {
+  id: string;
+  cardType: 'ArenaCategory' | 'ArenaTopic';
+  title: string;
+  subText?: string;
+  timestamp?: string;
+  count: number;
+  locked?: boolean;
+  categoryId?: string;
+}
+
+const StyledCategoryCard = css`
+  background-color: ${colors.background.default};
+  svg:nth-of-type(2) {
+    display: none;
+  }
+  &:hover,
+  &:focus-visible {
+    background-color: ${colors.background.lightBlue};
+    svg:nth-of-type(1) {
+      display: none;
+    }
+    svg:nth-of-type(2) {
+      display: block;
+    }
+  }
+`;
+
+const StyledTopicCard = css`
+  background-color: ${colors.background.lightBlue};
+  &:hover,
+  &:focus-visible {
+    background-color: ${colors.brand.lighter};
+  }
+`;
+
+const StyledCardContainer = styled(SafeLink)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: ${spacing.normal};
+  padding-right: ${spacing.medium};
+  border: 1px solid ${colors.brand.light};
+  border-radius: ${misc.borderRadius};
+  box-shadow: none;
+  > * > span {
+    text-decoration: underline;
+  }
+  &:hover,
+  &:focus-visible {
+    background-color: ${colors.brand.lighter};
+    > * > span {
+      text-decoration: none;
+    }
+  }
+`;
+
+const StyledTextContainer = styled.div`
+  margin-left: ${spacing.normal};
+  margin-right: auto;
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledHeader = styled(Text)`
+  color: ${colors.brand.primary};
+`;
+
+const StyledDescriptionText = styled(Text)`
+  padding-top: ${spacing.xsmall};
+  color: ${colors.text.primary};
+  ${mq.range({ until: breakpoints.mobileWide })} {
+    display: none;
+  }
+`;
+
+const StyledText = styled(Text)`
+  color: ${colors.text.primary};
+  padding-top: ${spacing.xsmall};
+`;
+
+const StyledCountContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: ${colors.text.primary};
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    display: none;
+  }
+`;
+
+const StyledCountDiv = styled.div`
+  ${fonts.sizes('18px', '24px')};
+  font-weight: ${fonts.weight.semibold};
+`;
+
+const StyledLeftIcon = styled(Icon)`
+  margin-right: ${spacing.normal};
+  min-width: 40px;
+  height: 40px;
+  color: ${colors.brand.primary};
+  ${mq.range({ until: breakpoints.mobileWide })} {
+    display: none;
+  }
+`;
+
+const StyledLockedIcon = styled(Icon)`
+  width: ${spacingUnit};
+  height: ${spacingUnit};
+  color: ${colors.brand.primary};
+`;
+
+const FolderFilledIcon = StyledLeftIcon.withComponent(Forum);
+const FolderOutlinedIcon = StyledLeftIcon.withComponent(ForumOutlined);
+const LockedIcon = StyledLockedIcon.withComponent(Locked);
```

Comment:
Tenker dette blir mer likt sånn vi gjør det i andre filer. Slipper man også bruken av `withComponent`. 
Bare bruk css attributen på `Forum` og `ForumOutlined` komponentene

```suggestion
const LeftIconCSS = css`
  width: ${spacing.large};
  height: ${spacing.large};
  color: ${colors.brand.primary};
  ${mq.range({ until: breakpoints.mobileWide })} {
    display: none;
  }
`;

const StyledLockedIcon = styled(Locked)`
  width: ${spacing.normal};
  height: ${spacing.normal};
  color: ${colors.brand.primary};
`;
```

Reply:
Jeg fjerna margin så legger vi heller på en `gap: ${spacing.normal}` på `StyledContainerCard`